articles/getting-started-with-quasar-framework/index.md

```diff
@@ -0,0 +1,295 @@
+```bash
+❯ Sass with SCSS syntax (recommended) 
+  Sass with SCSS syntax (recommended)
+  Sass with indented syntax (recommended)
+  Stylus (deprecated)
+  None (the others will still be available)
+```
+Select Auto-import-in-use Quasar Components and hit enter
+```bash
+? Pick a Quasar components & directives import strategy
+❯ * Auto-import in-use Quasar components & directives
+    - also treeshakes Quasar; minimum bundle size
+  * Import everything from Quasar
+    - not treeshaking Quasar; biggest bundle size    
+```
+Then press the spacebar to unselect the ESLint feature, we won't be needing it here. Hit enter.
+```bash
+❯ (*) ESLint (recommended)
+```
+Finally using arrow keys select yarn, this is what we will use during development, you can use npm if you like.
+```bash
+❯ Yes, use Yarn (recommended)
+```
+
+Navigate to the created `hello-world-app` and serve the app by running the commands:
+```bash
+$ cd hello-world-app
+$ quasar dev
+```
+After serving the app, it  opens the URL http://localhost:8080/#/ in your browser to view the app.
+You'll see quasar logo and the name quasar in the center of the page.
+
+Open the created `hello-world-app` folder in your editor of choice.
+
+Navigate to the `src` folder, open App.vue you'll see a div with and id of q-app
+```html
+<div id="q-app">
+```
+This is the where the quasar app is injected into the project using the id `q-app`.
+
+Navigate to the `src/layouts` folder and open `MainLayout.vue` file.
+
+Here we'll rename the app title to "Hello World App", if everything is okay you'll see the name change reflected on the toolbar at the top of the app.
+The toolbar title tag looks like this below
+```html
+<q-toolbar-title> Hello World App </q-toolbar-title>
+```
+You can delete the next div after the q-toolbar-title to remove the quasar version.
+```html
+<div>Quasar v{{ $q.version }}</div>
+```  
+The  MainLayout.vue file also contains navigation links for the app, this is where you'll put navigation links to your app while it grows bigger.
+
+For now, the links component which is the Essential link.vue file in the `components` folder is imported into the MainLayout.vue file. Data in linksData array is then looped through in the imported EssentialLink component and displayed to the left drawer of the app.
+
+Don't worry if you don't understand, you'll get a hang of it with more practice.
+
+Now navigate to the `src/pages` folder and open Index.vue.
+You can delete the whole image tag that looks like this.
+
+```html
+<img alt="Quasar logo" src="~assets/quasar-logo-full.svg">
+```
+This will delete the image and its name quasar from the center of the app. Leaving the app ready for other components
+
+### Familiarize with the structure of the components
+
+Here we're going to build a component. Just like Vuejs, Quasar uses single file components (SFC) structure. This has 3 parts; the HTML, SCRIPT(JS) and STYLE(CSS). If you have the know-how of building Vuejs apps this should be a breeze.
+
```

Comment:
Rewrite the sentence "This has 3 parts; the HTML, SCRIPT(JS) and STYLE(CSS)" using html tags. eg `<template>` (HTML)  to explain SFC. Don't forget to say that this tags are in a single file.
```suggestion
Here, we are going to build a component. Just like Vue.js, Quasar uses a single file components (SFC) structure. This has 3 parts; the HTML, SCRIPT(JS) and STYLE(CSS). If you have the knowledge of building Vue.js apps this should be a breeze.

```
Make the last sentence more formal.